### ISSUE TITLE
[Core-Extensions] MessageFactory Tests perform a redundant assert

### ISSUE THREAD

**Issue by stevengum97** [Microsoft/botbuilder-js#241]
This line of code appears in multiple tests:

```js
assert(activity.suggestedActions && activity.suggestedActions, `actions not returned`);
```

@Stevenic, is it supposed to be this?

```js
assert(activity.suggestedActions && activity.suggestedActions.actions, `actions not returned`);
```

#### Tests:

- [`should return suggestedActions().`](
https://github.com/Microsoft/botbuilder-js/blob/master/libraries/botbuilder-core-extensions/tests/messageFactory.test.js#L60)
- [`should return suggestedActions() with text.`](
https://github.com/Microsoft/botbuilder-js/blob/master/libraries/botbuilder-core-extensions/tests/messageFactory.test.js#L67)
- [`should return suggestedActions() with text and speak.`](
https://github.com/Microsoft/botbuilder-js/blob/master/libraries/botbuilder-core-extensions/tests/messageFactory.test.js#L75)
- [`should return suggestedActions() with text, speak, and inputHint.`](
https://github.com/Microsoft/botbuilder-js/blob/master/libraries/botbuilder-core-extensions/tests/messageFactory.test.js#L84)

**Comment by Stevenic** [comment]
Yes.. good catch. Want to fix and submit a PR? If not I'll add the fix to one of my future PR's.

**Comment by stevengum97** [comment]
Sure, I'll have a PR ready today.

**Comment by Stevenic** [comment]
Thanks :)
